vendor/istio.io/api/mesh/v1alpha1/config.proto

```diff
@@ -202,46 +210,13 @@ message MeshConfig {
   // and pass to sds server, which will be used to request key/cert eventually.
   // If both enable_sds_token_mount and sds_use_k8s_sa_jwt are set to true, enable_sds_token_mount(trustworthy jwt) takes precedence.
   // This isn't supported for non-k8s case.
-  bool sds_use_k8s_sa_jwt = 30;
+  bool sds_use_k8s_sa_jwt = 29;
 
   // The trust domain corresponds to the trust root of a system.
   // Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
   // Fallback to old identity format(without trust domain) if not set.
   string trust_domain = 26;
 
-  // Default service dependency settings for each workload in the
```

Comment:
Why is this being removed?

Reply:
cc @rshriram - this was removed in https://github.com/istio/api/pull/742.